greetings-maven/2.20.0-J11/org.xtext.example.mydsl.parent/pom.xml

```diff
@@ -8,6 +8,7 @@
 
 	<properties>
 		<xtextVersion>2.20.0-SNAPSHOT</xtextVersion>
+		<mwe2Version>2.11.1</mwe2Version>
```

Comment:
@lorenzobettini, do i see correct the var is not used in pure maven.
do we want this?